backend/tools/add_slot_from_db_for_schedule_admin.py

```diff
@@ -0,0 +1,22 @@
+import db.models as models
+from db.models import Slots, SlotsShema
+
+
+def add_slot_from_db_for_schedule_admin(start, end):
+    """Add new free slots in db for admin schedule
+    """
+    try:
+        slots = models.Slots(start_interval=start, end_interval=end)
+        models.db.session.add(slots)
```

Comment:
I wonder where do we start that transaction? I think there should be some call to start a transaction inside this function. Otherwise we can acciidentally commit or rollback some changes from other functions.

Reply:
it looks like they describe that here: https://flask-sqlalchemy.palletsprojects.com/en/2.x/queries/

Reply:
> I wonder where do we start that transaction? I think there should be some call to start a transaction inside this function. Otherwise we can acciidentally commit or rollback some changes from other functions.

I don't really understand what you mean?😀

Reply:
https://docs.sqlalchemy.org/en/13/orm/session_basics.html
I don't understand where the session starts. It looks like we have one session for the whole app in flask-sqlalchemy.

Let's see if we have troubles with that in the future.

Let's keep it as is for now.